packages/matter.js/src/crypto/Spake2p.ts

```diff
@@ -5,11 +5,17 @@
  */
 
 import BN from "bn.js";
-import { ec } from "elliptic";
 import { Crypto } from "./Crypto.js";
 import { ByteArray, Endian } from "../util/ByteArray.js";
 import { DataWriter } from "../util/DataWriter.js";
 
+// Lint complains about the next line but ts-node complains about importing
```

Comment:
Ääähmm ... my eslint do not complain!

Reply:
So in order to run the code gen without precompiling the project I wanted to run via ts-node.  Works pretty well after some fiddling but with elliptic in ES6 module mode (which is necessary for other reasons) ts-node complains that there is no named export in the library.  So I changed to a default import.  But then lint failed insisting that I should be using a named import.  Finally gave up and resorted to disabling lint.  I think I just pulled a muscle rolling my eyes.

Reply:
Provde "the"/"a" tsconfig? Because there is one tsconfig option for exactly that topic ... maybe we just need to use it

Reply:
I can revisit.  I did try the various tsconfigs.  If I remember correctly the ones that put it into CJS mode didn't work for other reasons.  But you do have other places using ts-node?

Reply:
I think we only have tsnode on matter-node level ... and in testing (jest/karma). could it be you need 

"esModuleInterop": true

??